src/Pelagos/Bundle/AppBundle/Resources/views/Menus/main.html.twig

```diff
@@ -34,7 +34,7 @@
                                             <span class="sf-sub-indicator"> »</span></span>
                         <ul class="sf-hidden" style="float: none; width: 13.5em;">
                             <li class="first odd "><a href="{{ path('pelagos_app_ui_account_default') }}">Account Request</a></li>
-                            <li class="middle even "><a href="{{ path('pelagos_app_ui_account_passwordreset') }}">Password Reset</a></li>
+                            <li class="middle even "><a href="{{ path('pelagos_app_ui_account_password') }}">Password Reset</a></li>
```

Comment:
We could change the verbage here to change/reset password, or base the verbage on whether or not they're logged in. I guess the safe bet is to see if the P.O. says anyting in testing. ;-)